mesos_master/tox.ini

```diff
@@ -17,6 +17,8 @@ passenv =
     DOCKER*
     COMPOSE*
 setenv =
+    1.0: MESOS_MASTER_RAW_VERSION=1.0.1
+    1.1: MESOS_MASTER_RAW_VERSION=1.1.0
```

Comment:
Too many vars & they may change, let's just not assert `patch`

Reply:
Hey @ofek , thanks for reviewing! If I remove `MESOS_MASTER_RAW_VERSION` and refactor the `test_metadata` function to assert the first two values of `MESOS_MASTER_VERSION`, would that address your suggestion? 

For example,

```
    major, minor = MESOS_MASTER_VERSION.split('.')[:2]
    version_metadata = {
        'version.scheme': 'semver',
        'version.major': major,
        'version.minor': minor
    }

    datadog_agent.assert_metadata('test:123', version_metadata)
    datadog_agent.assert_metadata_count(len(version_metadata) + 2)
```

I noticed this approach also being used in the mongo tests: https://github.com/DataDog/integrations-core/blob/673472adfc6af347a46bc7d82616852082fa8929/mongo/tests/test_mongo.py#L143-L150

Reply:
Yes please!